src/ldc/attributes.d

```diff
@@ -223,3 +223,23 @@ immutable weak = _weak();
 private struct _weak
 {
 }
+
+///Readability aliases for compute
```

Comment:
You should add that to the documentation of `@compute`. No example is given, no direct reference to `alsoProduceHostCode`.

You can make an enum class and retype `alsoProduceHostCode` to that. `alsoProduceHostCode` should be renamed if it is meant to be an `int`.
A simpler solution, but less nice:
```d
struct compute {
  enum deviceOnly = 0;
  enum hostAndDevice = 1;
  int codeProduction = deviceOnly;
}
```

By the way, you should check whether applying a UDA to a module declaration is already tested in dmd-testsuite. If there is no test for it, you should make sure it is added asap.